android/src/main/java/io/split/splitio/SplitMethodParserImpl.java

```diff
@@ -177,6 +182,15 @@ public void onMethodCall(String methodName, Object arguments, @NonNull MethodCha
                         mArgumentParser.getStringArgument(BUCKETING_KEY, arguments));
                 result.success(null);
                 break;
+            case SPLIT_NAMES:
+                result.success(mSplitWrapper.splitNames());
```

Comment:
Gaston, just in case, check if it is `names` or `splitNames`. In JS we have it as `names`, but maybe `splitNames` is the correct one.